src/install.js

```diff
@@ -104,6 +104,8 @@ async function main() {
     fileSource = fileSource.replace(/\/+$/g, '');
   }
 
+  let fileTemplate = argv['ft'] || argv['file-template'] || 'releases/{{install_version}}/resource.yaml';
```

Comment:
lets change the arg name to:
```suggestion
  let filePath = argv['fp'] || argv['file-path'] || 'releases/{{install_version}}/resource.yaml';
```